US-CBP/cbp-theme
# updated roboto-font-face library

#### What's this PR do?
- updates the Roboto-Font-Face Library to remove extranious fonts from the build 
#### Where should the reviewer start?
#### How should this be manually tested?
- run `npm install` of the project and verify only woff and woff2 files are in the fonts folder
#### Any background context you want to provide?
#### What are the relevant tickets?
- TCFU -1737
#### Screenshots (if appropriate)
#### Questions:
- Is there a blog post?
- Does the knowledge base need an update?
